### ISSUE TITLE
Determine security/access error when using list()

### ISSUE THREAD

**Issue by aaronksaunders** [angular/angularfire2#146]
if this is my code... and the user does not have access to this path, where should I expect to catch the access error?
```
var s = this.af.list('/users');
```
All of these approaches give back no results...
```
s.subscribe(
    () => { console.log("data") },
    () => { console.log("error") },
    () => { console.log("complete") }
)

s.catch(err => {
    console.log(err || 'No errors!')
    return err
});
```

**Comment by davideast** [comment]
I'm on the case! 🔍

**Comment by davideast** [comment]
Found the 🐛. Fixing. Thanks @aaronksaunders!

**Comment by aaronksaunders** [comment]
Thx, next time is there a better approach for submitting issues? Maybe a plnkr or something to show the actual issue?

**Comment by davideast** [comment]
@aaronksaunders This one was pretty simple, so it was totally fine without a plnkr. But plnkr's are always welcome!

Have it fixed in #145. Will ping you when it's out!

**Comment by aaronksaunders** [comment]
thx - great stuff
